test/e2e_node/util.go

```diff
@@ -394,19 +394,24 @@ func getCRIClient() (internalapi.RuntimeService, internalapi.ImageManagerService
 }
 
 // TODO: Find a uniform way to deal with systemctl/initctl/service operations. #34494
-func findRunningKubletServiceName() string {
-	stdout, err := exec.Command("sudo", "systemctl", "list-units", "*kubelet*", "--state=running").CombinedOutput()
-	framework.ExpectNoError(err)
-	regex := regexp.MustCompile("(kubelet-\\w+)")
-	matches := regex.FindStringSubmatch(string(stdout))
+func findRunningKubeletServiceName() string {
+	stdout, matches := kubeletStatus()
 	framework.ExpectNotEqual(len(matches), 0, "Found more than one kubelet service running: %q", stdout)
 	kubeletServiceName := matches[0]
 	framework.Logf("Get running kubelet with systemctl: %v, %v", string(stdout), kubeletServiceName)
 	return kubeletServiceName
 }
 
+func kubeletStatus() ([]byte, []string) {
```

Comment:
I feel `kubeletRunningStatus()` or `getKubeletRunningStatus()` can be better because this returns running Kubelet only.